cmd/directpv/main.go

```diff
@@ -94,7 +95,9 @@ func init() {
 	flag.Set("logtostderr", "true")
 	flag.Set("alsologtostderr", "true")
 
-	mainCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", kubeconfig, "Path to the kubeconfig file to use for Kubernetes requests.")
+	genericOptions = genericclioptions.NewConfigFlags(true)
+	genericOptions.AddFlags(mainCmd.PersistentFlags())
```

Comment:
This is not going to work. We use flags which colliding with `genericcliflags`. In fact we don't need support flags like `kubectl`.

Reply:
> We use flags which colliding with genericcliflags

Then use the flags from the flagset where they collide?  I'm happy to work with you to identify where this is and see if it's trivial to access it via the flagset.  I'm not sure if viper is able to access it as I haven't tested

> In fact we don't need support flags like kubectl.

As referenced by #884, your tool does not work for those that are working with multiple kubeconfig files very easily.  The flagset and its associated loaders provide a much more robust cluster and namespace selection, rather than having to juggle multiple kubeconfig directories.

Reply:
> We use flags which colliding with genericcliflags

Apologies -- looking at the test output, it's not that you're using the same flags, there's reused shortnames.  That is more of an issue.